docs/reference/config-params.md

```diff
@@ -5,196 +5,227 @@ title: Kev configuration reference
 
 # Configuration
 
-Kev leverages Docker Compose specification to configure and prepare an application for deployment in Kubernetes. Environment configuration lives in a dedicated docker compose override file, which automatically gets applied to the project's source sdocker compose files at `render` phase.
+Kev leverages the Docker Compose specification to configure and prepare an application for deployment in Kubernetes.
+
+## Project wide configuration
+
+This is optional, but if provided, lives in the base docker compose files (or at least one of the files) used to initialise the project.
+
+Project wide configuration will be applied against all environments unless a specific environment overrides a setting with its own value. 
+
+## Environment configuration
+
+Environment configuration lives in a dedicated docker compose override file. This automatically gets applied to the project's source docker compose files at the `render` phase.
+
+Any project wide configuration found will be overridden by environment specific values.  
 
 ### Component level configuration
 
 Configuration is divided into the following groups of parameters:
-* [Component](#-component)
-* [Workload](#-workload)
-* [Service](#-service)
-* [Volumes](#-volumes)
-* [Environment](#-environment)
+
+* [Component](#-component), part of the `x-k8s` docker compose service extension.
```

Comment:
perhaps instead of duplicating the same comment, we can put a top level item grouping these things together? Not sure if we need those extra comments here as the configuration settings will become apparent in their respective sections.

Reply:
Agreed, not really required as it will become obvious.